src/libsync/abstractnetworkjob.cpp

```diff
@@ -173,7 +173,7 @@ void AbstractNetworkJob::slotFinished()
     // get the Date timestamp from reply
     _responseTimestamp = _reply->rawHeader("Date");
 
-    if (_followRedirects) {
+    if (_followRedirects && reply()->operation() == QNetworkAccessManager::GetOperation) {
```

Comment:
@guruz Is this correct? Why is `operation` an attribute of the reply? Shouldn't it be part of the request?